util/rbac/rbac.go

```diff
@@ -273,12 +273,33 @@ func (a *argocdAdapter) LoadPolicy(model model.Model) error {
 			if line == "" {
 				continue
 			}
-			persist.LoadPolicyLine(line, model)
+			if err := loadPolicyLine(line, model); err != nil {
+				return err
+			}
 		}
 	}
 	return nil
 }
 
+// loadPolicyLine loads a text line as a policy rule to model.
+func loadPolicyLine(line string, model model.Model) error {
+	if line == "" || strings.HasPrefix(line, "#") {
+		return nil
+	}
+
+	reader := csv.NewReader(strings.NewReader(line))
+	reader.TrimLeadingSpace = true
+	tokens, err := reader.Read()
+	if err != nil {
+		return err
+	}
+
+	key := tokens[0]
+	sec := key[:1]
+	model[sec][key].Policy = append(model[sec][key].Policy, tokens[1:])
```

Comment:
Can we please add some clarification what `key` and `sec` actually represent in this context? Like, a little comment would be nice :)

Reply:
to be honest I don't know what is sec and key here :).  The function is a copy-paste from https://github.com/casbin/casbin/blob/master/persist/adapter.go#L24 with improved parsing that uses csv reader. Added comment to the function definition.